extensions/interactions/TextInput/TextInput.py

```diff
@@ -76,7 +76,7 @@ class TextInput(base.BaseInteraction):
         'default_value': 1,
     }, {
         'name': 'catchMisspellings',
-        'description': 'Catch Misspellings (Detect if answer is misspelled' +
+        'description': 'Catch Misspellings (Detect if answer is misspelled'
```

Comment:
Can you add the string into brackets? The next line is hard to differ from other keys.

Reply:
```
'description': (
    'Catch Misspellings (Detect if answer is misspelled '
    'and nudge the learner to correct the misspelling)'
),